graph/airdrops/mappings/common/campaign/lower-min-fee-usd.ts

```diff
@@ -0,0 +1,25 @@
+import { ethereum } from "@graphprotocol/graph-ts";
+import { Params } from "../../../helpers/types";
+import { Store } from "../../../store";
+
+export function handleLowerMinFeeUSD(event: ethereum.Event, params: Params.LowerMinFeeUSD): void {
+  const campaign = Store.Campaign.get(event.address);
+  if (campaign === null) {
+    return;
+  }
+
+  // Starting with v1.3, the constructor emits a TransferAdmin event.
+  if (params.previousMinFeeUSD.equals(params.newMinFeeUSD)) {
+    return;
+  }
```

Comment:
This code can be removed because of this check in the Solidity contracts:

```solidity
// Check: the new min USD fee is lower than the current min fee USD.
if (newMinFeeUSD >= currentMinFeeUSD) {
    revert Errors.SablierMerkleBase_NewMinFeeUSDNotLower(currentMinFeeUSD, newMinFeeUSD);
}
```

It is not possible to emit a `LowerMinFeeUSD` event with a `newMinFeeUSD` that is equal to `currentMinFeeUSD`.